src/theme/mui.ts

```diff
@@ -504,12 +505,18 @@ const theme = createTheme({
     MuiAlert: {
       root: {
         color: fontColor,
-        height: '48px',
         alignItems: 'center',
       },
       standardWarning: {
         backgroundColor: alertWarning,
       },
+      standardInfo: {
+        backgroundColor: infoColor,
+        borderRadius: '8px',
+        '& svg': {
+          color: secondary,
+        },
+      },
```

Comment:
Why are you overriding it globally instead of theming it locally?

Reply:
The other two alerts were also defined globally, and also set the colors.
Border radius is a bit too custom tho, I would agree it should probably go to the info bar component.